terraform/modules/ecs/monitoring.tf

```diff
@@ -0,0 +1,51 @@
+resource "aws_kms_key" "alerts" {}
+
+resource "aws_sns_topic" "alerts" {
+  name              = "${var.prefix}-alerts"
+  kms_master_key_id = aws_kms_key.alerts.arn
+}
+
+resource "aws_sns_topic_subscription" "alert_emails" {
+  for_each  = toset(var.alert_emails)
```

Comment:
Can we make this default to `monitoring-{deployment id}@polytomic.com`?

Reply:
Of course! Adding that now